### DISCUSSION TITLE
Current situation of the EPG repository

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#444] [Show and tell]
Hello,

As everyone know, this repository is in a dormant state since GitHub removed Actions from our repositories.
While their sanction is accurately taken (and we respect it), lots of persons have been asking for the return of the main functioning of the EPG system (being the file creation).

This is why iptv-org/iptv-org.github.io#425 has been created.

As of today, I partially take care of creating updated EPG files (as I only upload a few specific EPG files, cannot take care of all of them).

For now, we'll ask each and every person to **be patient** while we take a solution that will correspond to everyone's wantings, and make sure not to create any issue related to this.

**Comment by albertovm** [comment]
Integration with WebGrab+Plus could be a solution?

http://213.126.50.203
http://213.126.50.203/epg-channels

**Comment by Dr3sdan** [comment]
Thanks for update.

I think alot of users have had issues with getting consistent updates as one day we'll get a user-to-user chat but won't know what the project updates are.

So thank you for at least doing the bare minimum.

**Comment by BellezaEmporium** [comment]
> Integration with WebGrab+Plus could be a solution?
> 
> http://213.126.50.203 http://213.126.50.203/epg-channels

This is a freeware solution and so far we would prefer not relying on semi-paid resources to access any content.

**Comment by boeder9** [comment]
> Integration with WebGrab+Plus could be a solution?
> 
> http://213.126.50.203 http://213.126.50.203/epg-channels

WGP+ is theft. You pay a fee for premium access then pay an additional fee for individual guides AND you're limited to certain number of EPGs

**Comment by albertovm** [comment]
Is there any way to run updates locally? In my own server? Or only direct access to xml files here? Thanks.

**Comment by PopeyeTheSai10r** [comment]
> Is there any way to run updates locally? In my own server? Or only direct access to xml files here? Thanks.

See https://github.com/iptv-org/epg#how-to-use

**Comment by BellezaEmporium** [comment]
> Thanks for update.
> 
> I think alot of users have had issues with getting consistent updates as one day we'll get a user-to-user chat but won't know what the project updates are.
> 
> So thank you for at least doing the bare minimum.

Thank you for your comprehension.

As of now i'm still doing a specific number of those, but with @freearhey 's recent changes we might make it faster. Stay tuned :)

**Comment by albertovm** [comment]
> > Is there any way to run updates locally? In my own server? Or only direct access to xml files here? Thanks.
> 
> See https://github.com/iptv-org/epg#how-to-use

As I understood, this method is only for download and share files locally. Wrong?
I was asking a method to update epg data (each channel) locally.

**Comment by freearhey** [comment]
@albertovm This manual is exactly how to download the manual yourself from one of the supported websites.

I have already added a small clarification to the text to make it clearer: https://github.com/iptv-org/epg/commit/963de4fb5ef2b110ca4767b29296ba287c5a71fc
